modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -46,7 +51,8 @@ const generateItem = <T extends Item>(editor: Editor, item: T, format: string, e
 });
 
 const generateItems = <T extends Item>(editor: Editor, items: T[], format: string, extractText: (item: T) => string, onAction: (item: T) => void): Menu.ToggleMenuItemSpec[] =>
-  Arr.map(items, (item) => generateItem(editor, item, format, extractText, onAction));
+  Arr.map(items, (item) =>
+    generateItem(editor, item, format, extractText, onAction));
```

Comment:
Can we put this back on one line?

```suggestion
  Arr.map(items, (item) => generateItem(editor, item, format, extractText, onAction));
```

Reply:
I'd find it better on two, but sure.